scripts/gpu_usage.py

```diff
@@ -14,7 +14,7 @@
 import matplotlib.pyplot as plt
 
 # TODO: Probably want to move this to dmlutils.
```

Comment:
Yes I do eventually want it there, since this code does need to be used by another repo (see (1) in the description above). I have another ticket for that since i don't want to take time for that right now while we sprint towards our thanksgiving deadline.